metrics/definitions.go

```diff
@@ -57,6 +65,39 @@ var (
 		TagKeys:     []tag.Key{KeyName, KeyPeerID},
 		Aggregation: view.Sum(),
 	}
+
+	QuicConnsView = &view.View{
```

Comment:
Not sure if this is lower "uic" on purpose considering the others?

Reply:
Have fixed this.